apps/backend/src/transaction/entities/payment.entity.ts

```diff
@@ -86,7 +83,7 @@ export class PaymentEntity {
   // PCC - Internals
 
   @ManyToOne(() => PaymentMethodEntity, (pm) => pm.method)
-  @JoinColumn({ name: 'method' })
+  @JoinColumn([{ name: 'payment_method', referencedColumnName: 'method' }])
```

Comment:
Seemed redundant to have a 'method' as well as a 'payment_method'. Using the joined column for payment_method throughout the codebase. Using the payment_method.deposit_file_type to ease querying for cash payment types (using method not in ('AX', 'M', 'V', 'P') is less reliable since we have more TDI34 payment methods than in this list)

Reply:
I am not sure I understand. Both the tables now have the column `method` right?